src/frontend-proxy.ts

```diff
@@ -0,0 +1,95 @@
+import { getSubdomain, getPathname } from './url-utils'
+
+export const FRONTEND_DOMAIN = 'frontend-sooty-ten.now.sh'
+const API_ENDPOINT = 'https://api.serlo.org/graphql'
+const FRONTEND_ALLOWED_TYPES = ['Article', 'TaxonomyTerm']
```

Comment:
@Entkenntnis This variable contains the allowed types for which the frontend can be served. Which types shall I add here?

Reply:
Frontend also supports Video, Applet, Course and CoursePage

Reply:
and landing-page?

Reply:
IMHO: we should white-list the types one-by-one instead of everything at once (and integrate feedback on the way)